### PR TITLE
Fix audio RTL visual bugs

### LINKED ISSUE #421: RTL touchups
## Description
While the site still functions under RTL language direction, it looks extremely unpolished and will give a bad impression (in addition to the frustrating partial translations). What follows is a list of the individual places that need to be touched up to get the site "presentable" for RTL languages.

The following descriptions are all what it should be for RTL languages. For each fix, ensure that it does not cause a regression to the LTR languages.

- [x] #426
    - The navigation items should be aligned with the left side of the screen
    - The dropdown items should not have a large leading gap
    - Should the openverse logo be on the opposite side of the openverse brand text? cc @panchovm
- [x] #424
    - The search input and button should have the correct corners rounded
    - The "I want something I can..." checkboxes should have their spacing fixed to match the LTR language spacing
- [x] #431
    - The table has a double border on the left side
    - The table is missing the border between the Source and Domain column
    - The spacing of the "Suggest a new Source" button with the text and the icon is a little off compared to the LTR version (the left side of the button has too much padding)
    - Trailing slashes for all domains listed should be removed as they get automatically put at the front of the domain in RTL languages (`https://asia.si.edu/` gets visually transformed by the browser into `/https://asia.si.edu`)
- [x] #428
    - The search bar needs to have its button and input corners correctly rounded as well as its padding adjusted for RTL
    - The "safer browsing" flag icon needs to have its padding adjusted to match the LTR language padding
    - The scroll to top button should be moved to the opposite corner
    - License icons and their corresponding text need to have their separation padding adjusted for RTL
- [x] #435
    - "Go to image's website" button needs to have its icon padding adjusted
    - The attribution box and tabs need to have their borders adjusted to be on the correct side for RTL languages
    - The plain text attribution also needs to have its quotation marks corrected as currently they're being moved around by the browser into strange places
    - The license icon needs to have its padding adjusted against the associated text
    - The information table needs to have its padding adjusted as the "dimensions" text overlaps the listed dimensions
    - The leading edge of the information/reuse table needs to be fixed for RTL languages to align with the rest of the text in the result information
- [x] #512
    - The audio tags need to be adjusted for RTL languages, their borders are all over the place
- [x] #513
  -   The audio result license icon spacing compared to the track length needs to be adjusted, see [screenshots and description in the comments](https://github.com/WordPress/openverse-frontend/issues/428#issuecomment-974044511).

## Reproduction
The easiest way to reproduce is to set your language to an RTL language (like Arabic). I'm not sure what the best way to do this is, but the way I did it was to edit to `i18n_redirected` cookie value to be `ar`. I don't know if this cookie is always present, it might exist only because I tested the WordPress.org Openverse theme translations at one point. If that is the case, then follow [the instructions here](https://github.com/WordPress/wordpress.org/blob/master/wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/README.md) to set up the theme on a local WordPress install and set your language via wp-admin. Alternatively, manually edit the `dir` attribute on each page to be `rtl`. The attribute will be reset to `ltr` on each page load, so this is a very tedious option for testing as you'll have to edit the attribute every time you reload the page or navigate to a new one.

Because we will inevitably have partial translations down the line as well (for example when new strings are introduced or existing strings are updated), the site should look passable as RTL even with English strings, so for now the partial translations work in our favor as we fix these issues.

## Expectation

After fixing the issues above, the site should look polished in both LTR and RTL language directions.

For most of these the best solution might be to convert the underlying markup to use Tailwind CSS with the RTL ready start/end based styles.

## Environment

 - Device: 
 - OS: 
 - Browser: 
 - Version: 
 - Other info: 

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

### LINKED ISSUE #512: Single result page for Audio RTL touchups
The audio tags need to be adjusted for RTL languages, their borders are all over the place:

Captura de Tela 2021-12-13 às 09 51 03

### LINKED ISSUE #513: Audio search result RTL issues
The audio result license icon spacing compared to the track length needs to be adjusted:

Captura de Tela 2021-12-13 às 09 51 52

### PR DESCRIPTION
**Pull request by sarayourfriend**
## Fixes

Fixes #513 by @sarayourfriend
Fixes #512 by @sarayourfriend

Closes #421 as this will address the last two problems in that issue.

## Description

Fixes the RTL visual bugs outlined in the issues.

## Testing Instructions

Checkout this branch and run it with `npm run dev`. Then go to http://localhost:8443/audio/8071128a-bebc-4c24-823d-6e26a2e6dfe2 and verify that the two elements outlined in the issue look good in both RTL and LTR mode. Follow the instructions in [this issue](https://github.com/WordPress/openverse-frontend/issues/421) under the Reproduction headline for how to quickly test RTL and LTR locally.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [N/A] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/AudioTrack/layouts/Row.vue [hunk 1]
```diff
@@ -50,8 +50,8 @@
             </span>
           </div>
 
-          <div class="part-b inline-flex space-x-1">
-            <div v-if="isMedium">
+          <div class="part-b inline-flex">
+            <div v-if="isMedium" class="pe-1">
```
Comment (dhruvkb) [inline src/components/AudioTrack/layouts/Row.vue:54]: `space-x-` classes apply to all children as in
```css
.space-x-1 > * + * {
  margin-left: 0.25rem;
}
```
Replacing it with `pe-` classes will mean adding it to all children except the first. Alternatively we could use `gap-` classes on the flex itself?

Comment (sarayourfriend) [reply src/components/AudioTrack/layouts/Row.vue:54]: Leaving padding on the last element doesn't seem to introduce any visual problems that I could see. While not being a 1:1 replacement for `space-x-*` I think it's appropriate in this case? I guess the tailwindcss-rtl plugin doesn't cover the `space-<dir>-<unit>`.

`gap` seems like a better solution though. I'll update to that.

Comment (sarayourfriend) [reply src/components/AudioTrack/layouts/Row.vue]: Fixed!

**Review by zackkrida — APPROVED** [review]
LGTM!

**Review by dhruvkb — APPROVED** [review]

**Pull request merged**